numpy/core/numeric.py

```diff
@@ -1226,11 +1226,32 @@ def rollaxis(a, axis, start=0):
     a : ndarray
         Input array.
     axis : int
-        The axis to roll backwards.  The positions of the other axes do not
+        The axis to be rolled. The positions of the other axes do not
         change relative to one another.
     start : int, optional
-        The axis is rolled until it lies before this position.  The default,
-        0, results in a "complete" roll.
+        When ``start <= axis``, the axis is rolled back until it lies in
+        this position. When ``start > axis``, the axis is rolled until it
+        lies before this position. The following table describes how negative
+        values of ``start`` are interpreted:
+
+        .. table::
+           :align: left
+
+           +-------------------+----------------------+
+           |     ``start``     | Normalized ``start`` |
+           +===================+======================+
+           | ``-(arr.ndim+1)`` | raise ``AxisError``  |
+           +-------------------+----------------------+
+           | ``arr.ndim``      | 0                    |
```

Comment:
This row appears twice, yet is something different?

Reply:
```suggestion
           | ``-arr.ndim``     | 0                    |
```
Possibly? I don't remember where I wrote the original table.

Reply:
Ah, it was here: https://github.com/numpy/numpy/pull/15939#discussion_r406861923